lib/utils.js

```diff
@@ -0,0 +1,18 @@
+export function escapeRegexString(regexString) {
+  // See: http://stackoverflow.com/a/6969486/1463681
+  return regexString.replace(/[-[\]/{}()*+?.\\^$|]/g, '\\$&');
+}
+
+/**
+ * Used to determine how the pasted content is split.
+ * @returns {RegExp} Regular expression
+ */
+export function convertDelimitersToRegex(delimiters) {
```

Comment:
can you rename this function to getDelimitersForRegex` or `buildDelimitersFromRegex` or `createDelimitersFromRegex`as its creating the regex with delimiters passed in params and not converting the delimiters to regex ?

Reply:
I'm not sure I understand. You are taking the `delimiters` and turning (converting) them into a regex.

all of your names make it sound to me like you are creating the delimiters rather than the regex, whereas my name is much more clear as to what the input and output is

Reply:
I m sorry my bad. I made a typo here. I meant can you rename this function to `getRegexForDelimiters` or `buildRegexFromDelimiters` or `createRegexFromDelimiters` as you said its creating the regex from the delimiters.